recipes/grpc/all/conanfile.py

```diff
@@ -79,15 +79,23 @@ def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             tools.check_min_cppstd(self, 11)
 
-        if tools.is_apple_os(self.settings.os) and "arm" in self.settings.arch:
+        # Temporary workaround because the conan center CI/CD can't handle M1 Macs.
+        if hasattr(self, "settings_build") and tools.cross_building(self):
+            # If cross bulding we check settings build
+            settings = self.settings_build
+        else:
+            # otherwise just settings
+            settings = self.settings
+
+        if tools.is_apple_os(settings.os) and "arm" in settings.arch:
```

Comment:
```suggestion
        if hasattr(self, "settings_build") and tools.cross_building(self) and settings.os == "Macos" and "arm" in settings.arch:
```

Reply:
well actually it should be allowed again, protobuf on M1 is fine.

Reply:
Nice! That's great :)

Reply:
PR changed to just remove the check